src/lib/components/forms/form/form-components/text-area/text-area.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, Input } from '@angular/core';
+import { TerraPlacementEnum } from '../../../../../helpers';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { TextAreaInterface } from './text-area.interface';
+import { noop } from 'rxjs';
+
+@Component({
+    selector: 'tc-text-area',
+    templateUrl: './text-area.component.html',
+    providers: [
+        {
+            provide: NG_VALUE_ACCESSOR,
+            useExisting: TextAreaComponent,
+            multi: true
+        }
+    ]
+})
+export class TextAreaComponent implements ControlValueAccessor, TextAreaInterface {
+    /** If true, the textarea is not resizeable. Default false. */
+    @Input()
+    public set hasFixedHeight(fixedHeight: boolean) {
+        this._hasFixedHeight = fixedHeight;
+    }
```

Comment:
Es wäre hier einfacher einen normalen input daraus zu machen. Hier wird ja keine extra logik benötigt. Also
```suggestion
    public hasFixedHeight: boolean = false;
```
Damit würde dann sowohl die extra Property `_hasFixedHeight` als auch die Zeile zur Initialisierung im Constructor wegfallen.